modules/alloy/src/main/ts/ephox/alloy/api/ui/SplitFloatingToolbar.ts

```diff
@@ -87,6 +87,11 @@ const factory: CompositeSketchFactory<SplitFloatingToolbarDetail, SplitFloatingT
           FloatingToolbarButton.toggle(floatingToolbarButton);
         });
       },
+      toggleWithoutFocusing: (toolbar: AlloyComponent) => {
+        memFloatingToolbarButton.getOpt(toolbar).each((floatingToolbarButton) => {
+          FloatingToolbarButton.toggleWithoutFocusing(floatingToolbarButton);
```

Comment:
You can inline this. It might be a bit nicer:

```
memFloatingToolbarButton.getOpt(toolbar).each(FloatingToolbarButton.toggleWithoutFocusing)
```

Reply:
Thanks! Simplified [here](https://github.com/tinymce/tinymce/pull/8272/commits/6234a9905af406f5fa199f372df6408589fbf5b0).